### DISCUSSION TITLE
Wrong Password

### DISCUSSION THREAD

**Discussion by Thieberius** [GameServerManagers/LinuxGSM#3263] [Q&A] [answered]
Hi 
i installed the server with the script.
its showing up, i can add it to the favorites on steam,but if i want to connect its says wrong password.
i looked up the password in the start_server.sh but its the same i entered.

what am i doing wrong?

**Comment by wldwalker** [comment, marked as answer]
Would you be able to provide more infomation? What Distro are you using, what game server are you trying to run? While using LGSM you should **not** be editing "start_server.sh". 
See more information on how to configure your game server here: [https://docs.linuxgsm.com/configuration/linuxgsm-config](url)

**Reply by Thieberius** [reply]
i solved the problem, i was editing the wrong file...
so the whole time my server was without password aso... 

I was editing the start script, i thought the .x86 something comment was the one to edit to get a password working, set name etc....
